src/types/shared.ts

```diff
@@ -280,9 +281,11 @@ export interface SymbolLotSizeFilter {
 }
 
 export interface SymbolMinNotionalFilter {
-  filterType: 'MIN_NOTIONAL';
+  filterType: 'NOTIONAL';
```

Comment:
This is a breaking change, MIN_NOTIONAL is definitely there for futures:
https://binance-docs.github.io/apidocs/futures/en/#filters

and for spot as well:
https://binance-docs.github.io/apidocs/spot/en/#filters

Seems I don't have an interface yet for `filterType: 'NOTIONAL'`, I'd welcome a change to add it, but it should not be at the cost of removing the existing SymbolMinNotionalFilter interface. You can make a new one such as `SymbolNotionalFilter`, but please revert this breaking change.